third_party/souffle/parse_transform.cc

```diff
@@ -0,0 +1,311 @@
+          mk<ast::transform::FixpointTransformer>(mk<ast::transform::PipelineTransformer>(
+                  mk<ast::transform::ResolveAnonymousRecordAliasesTransformer>(),
+                  mk<ast::transform::FoldAnonymousRecords>())),
+          mk<ast::transform::PolymorphicObjectsTransformer>(), mk<ast::transform::SemanticChecker>(),
+          mk<ast::transform::ADTtoRecordsTransformer>(), mk<ast::transform::GroundWitnessesTransformer>(),
+          mk<ast::transform::UniqueAggregationVariablesTransformer>(),
+          mk<ast::transform::NormaliseMultiResultFunctorsTransformer>(),
+          mk<ast::transform::MaterializeSingletonAggregationTransformer>(),
+          mk<ast::transform::FixpointTransformer>(
+                  mk<ast::transform::MaterializeAggregationQueriesTransformer>()),
+          mk<ast::transform::ResolveAliasesTransformer>(), mk<ast::transform::RemoveTypecastsTransformer>(),
+          mk<ast::transform::RemoveBooleanConstraintsTransformer>(),
+          mk<ast::transform::ResolveAliasesTransformer>(), mk<ast::transform::MinimiseProgramTransformer>(),
+          mk<ast::transform::InlineRelationsTransformer>(),
+          mk<ast::transform::PolymorphicObjectsTransformer>(), mk<ast::transform::GroundedTermsChecker>(),
+          mk<ast::transform::ResolveAliasesTransformer>(),
+          mk<ast::transform::RemoveRedundantRelationsTransformer>(),
+          mk<ast::transform::RemoveRelationCopiesTransformer>(),
+          mk<ast::transform::RemoveEmptyRelationsTransformer>(),
+          mk<ast::transform::ReplaceSingletonVariablesTransformer>(),
+          mk<ast::transform::FixpointTransformer>(mk<ast::transform::PipelineTransformer>(
+                  mk<ast::transform::ReduceExistentialsTransformer>(),
+                  mk<ast::transform::RemoveRedundantRelationsTransformer>())),
+          mk<ast::transform::RemoveRelationCopiesTransformer>(), std::move(partitionPipeline),
+          std::move(equivalencePipeline), mk<ast::transform::RemoveRelationCopiesTransformer>(),
+          std::move(magicPipeline), mk<ast::transform::ReorderLiteralsTransformer>(),
+          mk<ast::transform::RemoveRedundantSumsTransformer>(),
+          mk<ast::transform::RemoveEmptyRelationsTransformer>(),
+          mk<ast::transform::AddNullariesToAtomlessAggregatesTransformer>(),
+          mk<ast::transform::PolymorphicObjectsTransformer>(),
+          mk<ast::transform::ReorderLiteralsTransformer>(), mk<ast::transform::ExecutionPlanChecker>(),
+          std::move(provenancePipeline), mk<ast::transform::IOAttributesTransformer>());
+
+  // Disable unwanted transformations
+  if (Global::config().has("disable-transformers")) {
+      std::vector<std::string> givenTransformers =
+              splitString(Global::config().get("disable-transformers"), ',');
+      pipeline->disableTransformers(
+              std::set<std::string>(givenTransformers.begin(), givenTransformers.end()));
+  }
+
+#if 0
```

Comment:
This uses some locals for profiling that I didn't need to set up when I ran the parser. I've now removed the block instead of disabling it with the preprocessor.